app/components/hiring_staff/vacancy_form_page_heading_component.rb

```diff
@@ -0,0 +1,50 @@
+class HiringStaff::VacancyFormPageHeadingComponent < ViewComponent::Base
+  delegate :current_organisation, to: :helpers
+  delegate :current_step, to: :helpers
+  delegate :total_steps, to: :helpers
+
+  def initialize(vacancy, session_vacancy_attributes)
```

Comment:
I would set a default value of an empty hash so that the code that follows is a bit nicer
```ruby
def initialize(vacancy, session_vacancy_attributes = {})
```

Reply:
Good idea